Source/OxyPlot/Graphics/ElementCollectionChangedEventArgs{T}.cs

```diff
@@ -0,0 +1,44 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="ElementCollectionChangedEventArgs{T}.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// <summary>
+//   The element collection changed event args.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// The element collection changed event args.
+    /// </summary>
+    /// <typeparam name="T">The type.</typeparam>
+    public class ElementCollectionChangedEventArgs<T> : EventArgs
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ElementCollectionChangedEventArgs{T}"/> class.
+        /// </summary>
+        /// <param name="addedItems">The added items.</param>
+        /// <param name="removedItems">The removed items.</param>
+        public ElementCollectionChangedEventArgs(IEnumerable<T> addedItems, IEnumerable<T> removedItems)
```

Comment:
For example, I subscribe to events of an Axis. Then when a user deletes an axis, I want to be able to unsubscribe to prevent memory leaks.